### PR TITLE
We have no time [Hackbar]

### PR DESCRIPTION
**Pull request by endrose**

### CODE REVIEW

File: _hackbars/endros.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+title: My First Hacktoberfest!
+author: Endros
+---
+
+Hello coders, my first time participation in hacktoberfest. excited to make this PR.
+
+
+---
+
+## We are no have time⏱
```
Comment (tasnimzotder) [inline _hackbars/endros.md]: hey @endrose Can you please change the line to `## We no have time⏱`?

**Comment by endrose** [comment]
Oke sir

**Review by tasnimzotder — APPROVED** [review]
👍

**Comment by tasnimzotder** [comment]
Thanks @endrose for your contributions :)

**Comment by endrose** [comment]
Thanks sir

**Pull request merged**